src/components/data/details/PermissionsPanel.js

```diff
@@ -267,7 +267,7 @@ function PermissionsTabPanel(props) {
             <Typography id={build(baseId, ids.SELF_PERMISSION)}>
                 {t("selfPermission", { permission: selfPermission })}
             </Typography>
-            {fetchResourcePermissionsKey && (
+            {selfPermission && selfPermission === Permissions.OWN && (
```

Comment:
```suggestion
            {selfPermission === Permissions.OWN && (
```
This should already result in `false` if `selfPermission` is falsey.